release_notes/topics/4_0_0_final.adoc

```diff
@@ -1,3 +1,15 @@
-= Test Feature
+= Client Scopes and suport for OAuth 2 scope parameter
 
-Some test feature.
\ No newline at end of file
+We added support for Client Scopes. Client Scopes replaced Client Templates. They allow more flexibility and better support for `scope` parameter,
+which we never properly supported until now! In relation to this, there are also some changes on consent screen. The items on consent
```

Comment:
Start new paragraph for the consent screen. 

"There are changes related to Client Scopes to the consent screen. The list on the consent screen is now linked to client scopes instead of protocol mappers and roles."

"See the documentation and migration guide for more details"